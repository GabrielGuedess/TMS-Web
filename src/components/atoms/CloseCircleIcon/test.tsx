import { render } from '@testing-library/react';

import { CloseCircleIcon } from '.';

describe('<CloseCircleIcon />', () => {
  it('should render the close circle icon correctly', () => {
    const { container } = render(<CloseCircleIcon />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
