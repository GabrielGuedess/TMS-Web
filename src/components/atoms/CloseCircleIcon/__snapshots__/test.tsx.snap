// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CloseCircleIcon /> should render the close circle icon correctly 1`] = `
<svg
  fill="none"
  height="24"
  viewBox="0 0 24 24"
  width="24"
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M12 22C17.5 22 22 17.5 22 12C22 6.5 17.5 2 12 2C6.5 2 2 6.5 2 12C2 17.5 6.5 22 12 22Z"
    stroke="currentColor"
    stroke-linecap="round"
    stroke-linejoin="round"
    stroke-width="1.5"
  />
  <path
    d="M9.16998 14.83L14.83 9.17"
    stroke="currentColor"
    stroke-linecap="round"
    stroke-linejoin="round"
    stroke-width="1.5"
  />
  <path
    d="M14.83 14.83L9.16998 9.17"
    stroke="currentColor"
    stroke-linecap="round"
    stroke-linejoin="round"
    stroke-width="1.5"
  />
</svg>
`;
